libr/asm/arch/avr/format.c

```diff
@@ -156,10 +156,34 @@ static int formatDisassembledOperand(avrDisassembleContext *context, char *strOp
 			fOptions.addressFieldWidth, context->longAddress);
 		break;
 	case OPERAND_IO_REGISTER:
-		retVal = sprintf(strOperand, "%s%02x",
-			OPERAND_PREFIX_IO_REGISTER,
-			dInstruction.operands[operandNum]);
+	{
+		char *current_register = NULL;
+		int is_register_found = 0;
```

Comment:
```suggestion
		bool is_register_found = true;
```

Reply:
I have already fixed it but I forgot to push. sorry.